gui/options.cpp

```diff
@@ -1276,12 +1273,10 @@ void OptionsDialog::addStatisticsControls(GuiObject *boss, const Common::String
 	uint16 width = g_system->getOverlayWidth() <= 320 ? 240 : 410;
 
 	for (uint16 idx = 0; idx < nMax ; idx++) {
-		Common::String key = info.stats[idx].id;
-		if (info.stats[idx].comment) {
-			key = info.stats[idx].comment;
-		}
+		const Common::StatDescription *descr = AchMan.getStatDescription(idx);
 
-		Common::String value = AchMan.getStatRaw(info.stats[idx].id);
+		Common::String key = descr->comment.empty() ? descr->id : descr->comment;
```

Comment:
There is imo no need to copy the string - so using a const ref here would be enough (`id` and `comment` are `String` instances now, no?).

Reply:
Thanks! Fixed at 312394e0802888ef62b983c9ab59ddd3578a7265